src/main/java/io/vertx/gsoc2018/qotd/QuoteOfTheDayVerticle.java

```diff
@@ -25,6 +29,38 @@ public void start(Future<Void> startFuture) throws Exception {
 
     jdbcClient = JDBCClient.createShared(vertx, jdbcConfig);
 
+    // web server setup
+    HttpServer server = vertx.createHttpServer();
+    Router router = Router.router(vertx);
+
+    router.route("/quotes").handler(routingContext -> {
```

Comment:
minor but `router.get` would be more specfic

Reply:
Yes, agree, it was changed to `router.get` in https://github.com/tsegismont/vertx-gsoc-2018-qotd/commit/162cb98ac70c1730f75b9e633dcdf08bad3b7cca when implementing POST /quotes

Reply:
:+1: